jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/view/jobs/JobModelListView.java

```diff
@@ -61,6 +63,10 @@
 @ViewDescriptor("job-model-list-view.xml")
 @LookupComponent("jobModelsTable")
 @DialogMode(width = "60em")
+@CssImport(
```

Comment:
Do not use this approach.
Use part type generator - https://demo.jmix.io/ui-samples/sample/data-grid-part-name-generator.
Put CSS file into `META-INF.resources.styles`. `module.properties` should contain `jmix.ui.export-styles` property (see Notifications as example).

Reply:
changed